src/components/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -33,42 +31,58 @@ export const BuyCreditsModal: FC<IProps> = () => {
       },
     },
   });
+  const [showIframe, setShowIframe] = useState(false);
   const credits = watch('credits', 0);
 
+  const iframeUrl = `https://ramptest.alchemypay.org/?apiKey=${process.env.NEXT_PUBLIC_API_KEY}&cryptoAddress=${process.env.NEXT_PUBLIC_CRYPTO_ADDRESS}&network=${process.env.NEXT_PUBLIC_NETWORK}`;
```

Comment:
This line is too long, let's get the env variables first
```suggestion
  const { 
    NEXT_PUBLIC_API_KEY: nextPublicApiKey,
    NEXT_PUBLIC_CRYPTO_ADDRESS: nextPublicCryptoAddress,
    NEXT_PUBLIC_NETWORK: nextPublicNetwork,
  } = process.env
  const iframeUrl = `https://ramptest.alchemypay.org/?apiKey=${nextPublicApiKey}&cryptoAddress=${nextPublicCryptoAddress}&network=${nextPublicNetwork}`;
```